app/uk/gov/hmrc/economiccrimelevyaccount/controllers/AccountController.scala

```diff
@@ -62,6 +61,10 @@ class AccountController @Inject() (
 
   private def getLatestObligation(obligationData: ObligationData): Option[ObligationDetails] =
     obligationData.obligations
-      .flatMap(_.obligationDetails.sortBy(_.inboundCorrespondenceDueDate)(Ordering[LocalDate].reverse))
+      .flatMap(
+        _.obligationDetails
+          .filter(record => record.status == Open)
+          .sortBy(_.inboundCorrespondenceDueDate)
```

Comment:
Do you need to specify the sort order? Previously it was (Ordering[LocalDate].reverse) but this has been dropped from this latest change.